sync/clients/sync.py

```diff
@@ -109,7 +110,16 @@ def delete_project(self, project_id: str) -> dict:
         return self._send(self._client.build_request("DELETE", f"/v1/projects/{project_id}"))
 
     def _send(self, request: httpx.Request) -> dict:
-        response = self._client.send(request)
+        try:
+            for attempt in Retrying(stop=stop_after_attempt(3), wait=wait_fixed(5)):
+                with attempt:
+                    response = self._client.send(request)
+                    if response.status_code == httpx.codes.SERVICE_UNAVAILABLE:
+                        # Doesn't need to be any particular exception, this is just to trigger the retry
+                        raise Exception()
+        except RetryError:
+            # If we max out on retries, then just bail and log the error we got
+            pass
```

Comment:
This will ignore any exceptions from the block won't it?

Reply:
I'm saying that without `reraise=True` exceptions that arise from within the `with attempt` block will be "wrapped" by a `RetryError` and ignored, and response won't be set. Not a big problem, but less than ideal IMO.

Reply:
ah, I see what you're saying! That's most likely true - let me give it a try to double check